contracts/Comptroller/Diamond/facets/SetterFacet.sol

```diff
@@ -526,7 +529,25 @@ contract SetterFacet is ISetterFacet, FacetBase {
         if (vTokenBorrowed != address(vaiController)) {
             ensureListed(markets[vTokenBorrowed]);
         }
-        isForcedLiquidationEnabled[address(vTokenBorrowed)] = enable;
+        isForcedLiquidationEnabled[vTokenBorrowed] = enable;
         emit IsForcedLiquidationEnabledUpdated(vTokenBorrowed, enable);
     }
+
+    /**
+     * @notice Enables forced liquidations for user's borrows in a certain market. If forced
+     * liquidation is enabled, user's borrows in the market may be liquidated regardless of
+     * the account liquidity. Forced liquidation may be enabled for a user even if it is not
+     * enabled for the entire market.
+     * @param borrower The address of the borrower
+     * @param vTokenBorrowed Borrowed vToken
+     * @param enable Whether to enable forced liquidations
+     */
+    function _setForcedLiquidationForUser(address borrower, address vTokenBorrowed, bool enable) external {
+        ensureAllowed("_setForcedLiquidationForUser(address,address,bool)");
```

Comment:
A side note for the VIP, I would grant Normal, Fast-track and Critical timelocks to perform this operation